openshift/temp-pod.yaml

```diff
@@ -0,0 +1,29 @@
+# I used this pod to put the TLS cert inside the "persistent" volume
+---
+kind: Pod
+apiVersion: v1
+metadata:
+  labels:
+    component: log-detective-website
+  name: log-detective-temp-pod
+spec:
+  containers:
+  - image: quay.io/jkadlcik/log-detective:latest
+    imagePullPolicy: Always
+    name: log-detective-temp-pod
+    resources:
+      limits:
+        cpu: "0.5"
```

Comment:
ditto

Reply:
I decided to drop the temp pod file because in the end it's not needed and one can create it on the fly easily. It also is only a bootstrapping problem, we will update the certs in future in the running pod (or even better, create a secret!)